src/modes/interOrderbook.ts

```diff
@@ -292,12 +292,23 @@ export async function findOpp({
                 // filter out the same owner orders
                 const opposingOrders = {
                     ...v,
-                    takeOrders: v.takeOrders.filter(
-                        (e) =>
-                            e.takeOrder.order.owner.toLowerCase() !==
-                            orderPairObject.takeOrders[0].takeOrder.order.owner.toLowerCase(),
-                    ),
+                    takeOrders: v.takeOrders
```

Comment:
is this change supposed to be on this PR?

Reply:
yes ofc